### ISSUE TITLE
create a role for external survey app

### ISSUE THREAD

**Issue by fherrick** [aamir-poswal/SurveyTrackerApp#35]
We need to have some security in survey app so only users that are invited will have the ability to create surveys.

Would like a prompt for a login when the url is used - http://216.65.214.89/#/

Please create a generic login and password and send to Muhammad and I. This will allow us to adminstrate the survey site.

We need the ability to invite users via email.
Survey sends invite via email, users login in using email and password (muhammad and I will create password)

User that are invited are now able to create surveys and send them out. User may only see surveys they have created.

Muhammad and Felicia can see all surveys using the generic login.

**Comment by aamir-poswal** [comment]
It is implemented, and ready for testing on dev. Please verify, and share your feedback.

Also I shared username/password through email.

**Comment by fherrick** [comment]
I was able to log in with no issues
![login](https://cloud.githubusercontent.com/assets/11509212/8320151/0488700e-19cc-11e5-8086-ad819c70e3b7.png)


Invited Leslie to survey app
![invite](https://cloud.githubusercontent.com/assets/11509212/8320165/2e1340e8-19cc-11e5-98b9-02cf39f1f9da.png)
![invitation](https://cloud.githubusercontent.com/assets/11509212/8320187/70c4c498-19cc-11e5-9a20-2538dcf737db.png)

![invitation](https://cloud.githubusercontent.com/assets/11509212/8320183/680699ee-19cc-11e5-92eb-9beac5aa30f0.png)

**Comment by fherrick** [comment]
are all the surveys gone? We are suppose to be able to see all the surveys using the generic login. 

Also after I created a use and sent invite I was not able to get back to the landing page or create a survey. I clicked create survey and nothing happened.

**Comment by fherrick** [comment]
are all the surveys gone? We are suppose to be able to see all the surveys using the generic login.

Also after I created a use and sent invite I was not able to get back to the landing page or create a survey. I clicked create survey and nothing happened.

Only able to get to All Survey page by logging off and logging back in.
![landing page](https://cloud.githubusercontent.com/assets/11509212/8320264/69ff95e2-19cd-11e5-9a83-35e9189b20b4.jpg)
